packages/components/src/components/select-native/select-native.jsx

```diff
@@ -29,6 +39,7 @@ const SelectNative = ({
     suffix_icon,
     use_text,
     value,
+    hide_disabled,
```

Comment:
Try to stick to the [styling guide](https://github.com/binary-com/deriv-app/blob/master/docs/JavaScript/README.md#naming-conventions-boolean) for this project by naming it something like below:

```suggestion
    should_hide_disabled_options
```

Above suggestion will also provide more context to the next developer.